go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/optimizely/go-sdk
+
+go 1.12
```

Comment:
Should this be 13?

Reply:
I think we should stay with 1.12 for now, 1.13 has been just released. We should wait and see if there are no major problems reported by go community.